### PR TITLE
[libmorton] Updated version to 0.2.11

### PR DESCRIPTION
**Pull request by Forceflow**
Updated libmorton from 0.2.8 to 0.2.11, which is the newest version. I am the maintainer of libmorton.

- [ v] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)
- [ v] SHA512s are updated for each updated download
- [n/a ] The "supports" clause reflects platforms that may be fixed by this new version
- [ n/a] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ n/a] Any patches that are no longer applied are deleted from the port's directory.
- [ ]v The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [ v] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by Forceflow** [comment]
@microsoft-github-policy-service agree

File: ports/libmorton/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO Forceflow/libmorton
-    REF v0.2.8
-    SHA512 7497203573965337F1216DB729D3F6806278B8DDFF539E8D01EF83FB3EAB50D5FF90C965CF5EA3A76464CF6F279C000434D2111B37D3D368834F7EF20959E790
+    REF v0.2.11
```
Comment (Cheney-W) [inline ports/libmorton/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

**Comment by Cheney-W** [comment]
- [ ] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.

##[error]Found the following errors:
    Error: While reading versions for port libmorton from file: C:\a\1\s\versions\l-\libmorton.json
       Version `0.2.11` was not found in versions file.
       Run:

           vcpkg x-add-version libmorton

       to add the new port version.

**Comment by Forceflow** [comment]
Fixed database, my apologies.

**Comment by Cheney-W** [comment]
Please run below command when you changed the port's file again:
```
           .\vcpkg x-add-version libmorton --overwrite-version
           git add versions
           git commit -m "Update version database"
```

**Comment by Forceflow** [comment]
Should be good now. Thanka for the help, @Cheney-W

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**